exist-core/src/main/java/org/exist/storage/BrokerPool.java

```diff
@@ -1547,7 +1519,7 @@ public void triggerSync(final Sync syncEvent) {
     //TOUNDERSTAND (pb) : synchronized, so... "schedules" or, rather, "executes" ?
     public void triggerSystemTask(final SystemTask task) {
         final State s = status.getCurrentState();
-        if(s == State.SHUTTING_DOWN) {
+        if(s == State.SHUTTING_DOWN_MULTI_USER_MODE || s == State.SHUTTING_DOWN_SYSTEM_MODE) {
```

Comment:
` isShuttingDown()`

Reply:
I need the state to be consistent, so we just get the state once and use it here and below to check if the database is shutdown.

If I called `isShuttingDown()` it would make a second call to `status.getCurrentState` and that would not be consistent with `s`.

Reply:
but... it has the same code?
![image](https://user-images.githubusercontent.com/1700062/105202049-db966300-5b41-11eb-90e5-41e3d5b48918.png)

Reply:
ah sorry Now I see it